game/theater/theatergroundobject.py

```diff
@@ -481,18 +502,77 @@ def __init__(
             dcs_identifier="AA",
             sea_object=False,
         )
-        # Set by the SAM unit generator if the generated group is compatible
-        # with Skynet.
-        self.skynet_capable = False
+        self.aa_ranges: List[AirDefenseRange] = [aa_range]
+
+    # Returns the name of one single group
+    # It also changes the group name to a skynet readable
```

Comment:
A skynet readable what?

Also, this and the other function/class level docs should be docstrings.